lints/lint_issuer_multiple_attr_in_rdn.go

```diff
@@ -0,0 +1,45 @@
+// lint_issuer_multiple_attr_in_rdn.go
+
+package lints
+
+import (
+	"encoding/asn1"
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zcrypto/x509/pkix"
+	"github.com/zmap/zlint/util"
+)
+
+type IssuerRDNHasMultipleAttribute struct {
+	// Internal data here
+}
+
+func (l *IssuerRDNHasMultipleAttribute) Initialize() error {
+	return nil
+}
+
+func (l *IssuerRDNHasMultipleAttribute) CheckApplies(c *x509.Certificate) bool {
+	return true
+}
+
+func (l *IssuerRDNHasMultipleAttribute) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	var issuer pkix.RDNSequence
```

Comment:
It's already parsed, but pkix.go discards any attribute beyond the first. We don't want to store anything beyond the first, because they are not adhering to the standard.